components/nexusoperations/workflow/commands_test.go

```diff
@@ -567,8 +561,6 @@ func TestHandleCancelCommand(t *testing.T) {
 		})
 		require.NoError(t, err)
 		require.Equal(t, 2, len(tcx.history.Events)) // Only scheduled event should be recorded.
-		crAttrs := tcx.history.Events[1].GetNexusOperationCancelRequestedEventAttributes()
-		require.Equal(t, event.EventId, crAttrs.ScheduledEventId)
```

Comment:
I think cancel requested should be recorded. Why remove this check?

Reply:
Fixed